core/store/migrate/migrations/0229_move_keyvalue_store_val_to_bytes.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
```

Comment:
@ettec You may have to regenerate this -- I think migration 0229 has been merged at this point.